app/src/main/java/at/roteskreuz/stopcorona/model/repositories/ExposureNotificationRepository.kt

```diff
@@ -64,6 +67,8 @@ interface ExposureNotificationRepository {
      * Refresh the current app status regarding the observe changes in [observeAppIsRegisteredForExposureNotifications].
      */
     fun refreshExposureNotificationAppRegisteredState()
+
+    fun getTemporaryExposureKeyHistory(): Task<MutableList<TemporaryExposureKey>>
```

Comment:
We use this repository to have a reasonable wrapper for the client. Please do not introduce new async way how to communicate with it. Use coroutines or rx observables if needed.

Reply:
as this is only a debug view to expose the framework interaction, I'm not doing anything with it yet. It will be removed in the next tickets